gravity-forms/gw-coupons-exclude-products.php

```diff
@@ -102,10 +103,11 @@ function output_script() {
 
 				function getExcludedAmount( formId ) {
 
-					var excludeFields = gf_global.gfcep[ formId ],
-						amount        = 0;
+					var excludeFields               = gf_global.gfcep[ formId ].exclude_fields,
+						excludeFieldsWithoutOptions = gf_global.gfcep[ formId ].exclude_fields_without_options,
+						amount                      = 0;
 
-					if( ! excludeFields ) {
+					if( ! excludeFields && ! excludeFieldsWithoutOptions ) {
```

Comment:
Should this be `||`?

Or in other words, shouldn't the calculations for `excludeFields` and `excludeFieldsWithoutOptions` be independant and not matter if the other exists or not?

Reply:
Followup: this question can be ignored if `excludeFields` and `excludeFieldsWithoutOptions` default to empty arrays 🙂